data/incipias/first contact.txt

```diff
@@ -0,0 +1,287 @@
+	on offer
+		conversation
+			`As the Heliarch have a relationship to you feel not really like friendship but something in that direction, you decide to ask them for a prove of this relationship, to give to the Hai.`
+			`    You approach a nearby Heliarch and tell him you want to ask the government about a personal matter. He gives you directions to an office that handles maters like yours. You thank the Heliarch and make your way to the office.`
+			`    When you arrive there, you end up in a pretty long waiting line. It seems that the Heliarch are never short of work. You patiently line yourself up in the line and get ready to spend some time in the line.`
+			`    After some time has passed it is finally your turn to tell your matter. You explain that you would like to have something that displays the good relationship between the Coalition and you. The Heliarch is quite surprised and says: "An interesting request of yours that is. Why you want this, may I ask?"`
+			choice
+				"I am gathering proves of friendship from all over the galaxy."
+					goto cages
+				"I want to show that species do not have to be in cages."
+			`    The Heliarch seems very happy when you say that, he says "A good reason that is, here you wait, while some people above me about this I ask." He turns around and goes away.`
+				goto c1
+			label cages
+			`    "An unusual project that is, what the reason for this is?"`
+			choice
+				"I am a passionate collector."
+				"I want something to remember you, to remember peace is possible."
+			`    "Yor  project understand I can. The people over me about your request let me ask." He turns around and goes away.`
+			label c1
+			`    When noises in the line behind you are growing louder he finally comes back. With him he carries a small box. "A decent gift for you I got." He opens the box and in there a small gem like no one you have seen before is laying, reflecting the light in green, red and blue shades. "This gem a fusion of three others it is, an emerald for the Saryds, a ruby for the Kimek and a sapphire for the Arachi. A fusion of our three species it is, you could say. This gem representing our species best it is. You appreciate this gift I hope." You are really thankful for this gift and thank the Heliarch a lot. But then you have to leave because there is still a line behind you.`
+				accept
+
+mission "Incipias: First Contact IV"
+	landing
+	name "A Hai delegation"
+	to offer
+		has "event: Hai preparation time"
+	source Mirrorlake
+	destination "Spec Inci"
+	on offer
+		conversation
+			`At the spaceport you see a small group of Hai diplomats, one of them is familiar to you from the embassy. You walk towards them and greet them. One of them says, "I hope for a save journey on your ship Captain." You answer, "The system we are travlling to is not far from here." He says, "Than not more is to be discussed, bring us to the Quarg ringworld, so that we can talk to them." You agree and get your ship ready.`
```

Comment:
```suggestion
			`At the spaceport you see a small group of Hai diplomats, one of them is familiar to you from the embassy. You walk towards them and greet them. One of them says, "I hope for a safe journey on your ship Captain." You answer, "The system we are travelling to is not far from here." He says, "Then no more is to be discussed, bring us to the Quarg ringworld, so that we can talk to them." You agree and get your ship ready.`
```